.github/workflows/bloat.yaml

```diff
@@ -62,10 +65,20 @@ jobs:
         name: Build base
         id: build_base
         run: |
-          make WEBASSETS_SKIP_BUILD=1 BUILDDIR=base_build binaries
+          BUILD_SECRET=FAKE_SECRET make WEBASSETS_SKIP_BUILD=0 BUILDDIR=base_build ensure-webassets binaries
```

Comment:
```suggestion
          BUILD_SECRET=FAKE_SECRET make BUILDDIR=base_build ensure-webassets binaries
```

Reply:
`make full` sets this explicitly, I felt like doing the same here helped make it explicit that we also want to include the webassets as part of this test.